<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenWidget</class>
 <widget class="QWidget" name="ScreenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_addr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>操作设定</string>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>跳转时注销用户</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ScreenIndex">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>72</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>画面名称：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_operType">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>跳转到下一个画面</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>跳转到指定画面</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>打开窗口</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>关闭窗口</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>返回上一个画面</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>32</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>操作类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_effect">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>正常显示</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>层叠</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>淡入淡出</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>平移</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>旋转</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>32</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>切换效果：</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
